src/components/Score.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+
+class score extends Component {
+
+    scoreStatus = () => {
```

Comment:
`const {score}=this.props`
then you would just have to write score in the rest of the function